NEWS.md

```diff
@@ -179,6 +179,9 @@ the imputation functions, e.g., `derive_vars_dtm()`. (#2222)
 - All "Example Script(s)" sections in the User Guide vignettes were updated to point the user towards using `use_ad_template("ADaM")` rather 
   than linking to the template in the code repository. (#2239)
 
+- Handling of `NA` values was added to the documentation of the `order` argument
+for all functions. (#2230)
```

Comment:
Can #2257 also be referenced since this issue spurred the discussion?

Reply:
Yes, good point.